slsim/Pipelines/sl_hammocks_pipeline.py

```diff
@@ -82,6 +100,82 @@ def __init__(self, slhammocks_config=None, sky_area=None, cosmo=None):
             table = table_translator_for_slsim(table, cosmo)
 
             self._pipeline = table
+        skypy_config = os.path.join(module_path, "data/SkyPy/slhammock_skypy.yml")
+        with open(skypy_config, "r") as file:
+            content = file.read()
+        redshift_array = list(self._pipeline["z"].value)
+        old_z = "z: PLACEHOLDER_Z"
+        new_z = f"z: {redshift_array}"
+        content = content.replace(old_z, new_z)
+        stellar_mass_array = list(self._pipeline["stellar_mass"].value)
+        old_stellar_mass = "stellar_mass: PLACEHOLDER_MASS"
+        new_stellar_mass = f"stellar_mass: {stellar_mass_array}"
+        content = content.replace(old_stellar_mass, new_stellar_mass)
+
+        if cosmo is not None:
+            if cosmo is default_cosmology.get():
+                pass
+            else:
```

Comment:
could this be a separate utility routine with documentation of what is happening here?